accounts-db/src/accounts_db.rs

```diff
@@ -4309,6 +4307,34 @@ impl AccountsDb {
             .fetch_add(time.as_us(), Ordering::Relaxed);
     }
 
+    pub(crate) fn unref_shrunk_dead_accounts<'a>(
+        &self,
+        unrefed_pubkeys: impl Iterator<Item = &'a Pubkey>,
+        slot: Slot,
+    ) {
+        self.accounts_index.scan(
+            unrefed_pubkeys,
+            |pubkey, slot_refs, _entry| {
+                // pubkeys in `unrefed_pubkeys` were unref'd in `shrink_collect` above under the assumption that we would shrink everything.
```

Comment:
doh. this is wrong. I copied this code probably. These keys are to be unref'd.

Reply:
yeah. fixed.